src/task.c

```diff
@@ -1221,8 +1279,7 @@ connections_write (AppData     *app_data,
                              app_data->cancellable,
                              NULL);
 
-    section = g_strdup_printf ("offsets_%s", task->task_id);
-    restraint_config_set (app_data->config_file, section, path, NULL, G_TYPE_UINT64, *offset);
+    (void) task_config_set_offset (app_data->config_file, task, path, *offset, NULL);
```

Comment:
Not sure why rc and error propagated and not used.

Reply:
Because it's not the intent of this PR to change the error handling in this function. The error is is ignored as it was in the `restraint_config_set` call.